src/libraries/System.IO.FileSystem/tests/File/ReadWriteAllBytes.cs

```diff
@@ -106,7 +107,7 @@ public void WriteToReadOnlyFile()
             try
             {
                 // Operation succeeds when being run by the Unix superuser
-                if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && geteuid() == 0)
+                if ((!RuntimeInformation.IsOSPlatform(OSPlatform.Windows) && !RuntimeInformation.IsOSPlatform(OSPlatform.Browser))  && geteuid() == 0)
```

Comment:
Remove extra set of parenthesis with short circuiting here as well as in the five files below?